helmfile/charts/networkpolicy-service/templates/opensearch/master.yaml

```diff
@@ -120,6 +120,10 @@ spec:
             cidr: {{ $ip }}
         {{- end }}
       {{- else }}
+        {{- range $ip := .Values.global.scIngress.ips }}
```

Comment:
Where there issues in opensearch as well?

Reply:
Not that I've experienced myself, but since this rule builds on the same principle it will have the same issue whenever that path is hit.